.github/workflows/main.yml

```diff
@@ -82,3 +82,33 @@ jobs:
         else
           echo "No changes; skip commit"
         fi
+
+  json-spec:
+    name: Generate and upload json-spec
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Minner om at `v2` er ute

Reply:
Jepp, brukte `v1` for å være konsekvent med resten av workflowen. Kan lage en annen PR for å oppgradere alle jobsa

Reply:
Skjønner, enig med at egen PR er en god idé.